openqabot/types/incident.py

```diff
@@ -144,3 +148,31 @@ def contains_package(self, requires: List[str]) -> bool:
                 if package.startswith(req) and package != "kernel-livepatch-tools":
                     return True
         return False
+
+    def has_failures(self, token) -> bool:
+        results = requests.get(
+            QEM_DASHBOARD + "/api/jobs/incident/" + str(self.id), headers=token
+        ).json()
+
+        failed_jobs = [res for res in results if res["status"] != "passed"]
```

Comment:
Good catch

Reply:
I assume https://github.com/openSUSE/qem-bot/blob/9872fbc88ccc14968f066f54b8565049754541bc/openqabot/utils.py#L57 has been called already so that passed and soft failed are both treated the same. In openQA we call those "ok results"